pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1437,3 +1495,31 @@ func isDeviceOpened(deviceToDetach AttachedVolume, mounter mount.Interface) (boo
 	}
 	return deviceOpened, nil
 }
+
+func isMigrated(vpm *volume.VolumePluginMgr, spec *volume.Spec) bool {
```

Comment:
add TODO comment that this should contain logic for checking CSINodeInfo for Kubelet migration status